idlegame/nanobots.py

```diff
@@ -78,28 +78,32 @@ def get_current_action(self, event = None) -> str:
             return f"Performing idle action: {self.idle_action}"
         return "IDLE"
 
-def handle_nano(player: AutosavedPlayer, *args, **kwargs) -> None:
-    """Write a new nanobot.
-
-        Usage:
-            nano [--type <nanotype>] [--name <name>] [-y]
+def interactive_nanobot_logic():
+    """Interactive text editor for creating nanobot logic, allowing edits to previous lines."""
+    lines = []
+    print("Write the logic for your nanobot (type 'done' on a new line to finish, 'edit X' to edit line X):")
+
+    while True:
+        line = input()
+        if line == 'done':
+            break
+        elif line.startswith('edit'):
+            try:
+                line_num = int(line.split()[1]) - 1
+                if 0 <= line_num < len(lines):
+                    print(f"Editing line {line_num + 1}: {lines[line_num]}")
+                    lines[line_num] = input("New content: ")
```

Comment:
This is not interactive; the objective is to be able to scroll through a multiline input and edit each line instead of editing on a per-line basis.